src/ddmd/errors.d

```diff
@@ -19,6 +19,16 @@ import ddmd.root.outbuffer;
 import ddmd.root.rmem;
 import ddmd.console;
 
+/**********************
+ * Color highlighting to classify messages
+ */
+enum Classification
+{
+    error = Color.red,          // for errors
+    gagged = Color.magenta,     // for gagged errors
+    warning = Color.yellow,     // for warnings
+    _deprecation = Color.blue,  // for deprecations
```

Comment:
@ZombineDev: how about adding this to the [D Style guide](http://dlang.org/dstyle.html)? The [Vibe.d](https://vibed.org/style-guide) already specifies this to some extent..